app/frontend/actions.ts

```diff
@@ -729,14 +696,6 @@ export default (store: Store) => {
     })
   }
 
-  const selectAdaliteStakepool = (state: State): void => {
```

Comment:
Was there any reason behind this? It seems to me that it is just fine to remove this.

Reply:
I thinks its fine as well.